packages/lib/services/plugins/utils/manifestFromObject.ts

```diff
@@ -31,6 +31,17 @@ export default function manifestFromObject(o: any): PluginManifest {
 		return o[name];
 	};
 
+	const getScreenshots = (defaultValue: Screenshot[] = []): Screenshot[] => {
+		if (!o.screenshots) {
+			return defaultValue;
+		} else {
+			for (let i = 0; i < o.screenshots.length; i++) {
+				if (typeof o.screenshots[i] !== typeof Screenshot) throw new Error('Field must be a screenshot object');
```

Comment:
why would you check the type of this object?? This is an object parsed from JSON - it has no type... You're just comparing `"object" !== "object"` here.

Reply:
Thanks for explaining it to me. I removed them now.

Reply:
But you need to be a lot more thorough with the way you test your code. You shouldn't assume that a particular line does what you think it does without checking.

In that particular case if you had done `console.info(typeof o.screenshots[i])` you would have seen that check wasn't useful.

Reply:
Thanks. I'll keep that in mind.